Riot/Modules/Room/VoiceBroadcast/VoiceBroadcastService.swift

```diff
@@ -0,0 +1,268 @@
+// 
+// Copyright 2022 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Foundation
+
+/// Voice Broadcast settings.
+@objcMembers
+final class VoiceBroadcastSettings: NSObject {
+    static let eventType = "io.element.voice_broadcast_info"
+    
+    static let defaultChunkLength = 600
```

Comment:
Don't you think this config should be moved in the BuildSettings with the `voiceBroadcastEnabled` flag?
I don't know if there is a preference for this kind of config definition.
This may be a question for the second reviewer